proxy/src/schema/mod.rs

```diff
@@ -87,12 +87,13 @@ impl Mutation {
         ctx: &Context,
         domain: String,
         name: String,
+        pid: Option<String>,
```

Comment:
```suggestion
        librad_id: Option<String>,
```

`pid` is otherwise too ambiguous.